### DISCUSSION TITLE
I want to use my tablet driver outside osu!lazer, how do I do that?

### DISCUSSION THREAD

**Discussion by oijdfdg** [ppy/osu#13550] [Q&A] [answered]
I want to use my wacom drivers instead of the built-in ones.
I tried: unchecking the tablet switch in the settings

**Comment by peppy** [comment, marked as answer]
You might need to turn "Enhance cursor precision" on or off, depending on what weird method your external drivers are using to update the mouse cursor position.

**Reply by oijdfdg** [reply]
I turned off the "High precision mouse" and it worked
